### DISCUSSION TITLE
[RFC] Adding JIT mode for Tailwind recipe

### DISCUSSION THREAD

**Discussion by Anjianto** [blitz-js/blitz#2303] [RFCs] [answered]
# Problem

Working with tailwind css in default mode is so hard and make it slow in Development moreover high end PC/Laptop. It will make slow in starting dev server, in web browser, Dev Tools, etc. And the main problem with default tailwind mode is we cannot use arbitary value like w-[71.5px], which sometime it can be usefull.

# Solution

1. Working with inline css

    First solution is use it inline css. Which is the general solution but it not so fun working like this.

2. use it css module

   Use Vanilla CSS make it our CSS globally. So, to prevent we use css module instead Vanilla CSS.

3. Use CSS in JS

    This Solution can be fun. But it will add new dependencies for use it, Which can be not so fun again.

4. Use it Tailwind JIT

    The last solution for me is use Tailwind JIT. Which currently preview mode aka not currently stable until take v3. We can use arbitary value like i say before.

**Comment by flybayer** [comment, marked as answer]
Yeah we should update the tailwind recipe to use JIT! Want to make a PR?

**Reply by Anjianto** [reply]
Oh Yeah, i will make PR right now
